test/libsolidity/syntaxTests/types/too_small_negative_numbers.sol

```diff
@@ -0,0 +1,4 @@
+contract C {
+  fixed8x80 a = -1 / (1e9 ** 10);
```

Comment:
`fixed8x80 a = -1e-100`; should also work.

Reply:
Your's is more minimal (and working), so I've changed the test.